### PR TITLE
Add CC and BCC (with multiple email support) to Send Email automation action step

### PR DESCRIPTION
**Pull request by melohagan**
## Description
You can now use CC and BCC in email automations.

Addresses: 
- https://github.com/Budibase/budibase/issues/4385

## Screenshots
Screenshot 2022-09-21 at 16 10 00

Screenshot 2022-09-21 at 16 10 26

Screenshot 2022-09-21 at 16 11 01

### CODE REVIEW

File: packages/worker/src/api/routes/global/email.js [hunk 1]
```diff
@@ -12,7 +12,13 @@ function buildEmailSendValidation() {
...
     email: Joi.string().email({
       multiple: true,
-      }),
+    }),
+    cc: Joi.string().email({
+      multiple: true,
+    }),
+    bcc: Joi.string().email({
+      multiple: true,
+    }),
```
Comment (Rory-Powell) [inline packages/worker/src/api/routes/global/email.js:21]: should these have `.optional()` appended?

Comment (melohagan) [reply packages/worker/src/api/routes/global/email.js]: Nice catch! I tried optional but it doesn't change the behaviour. Changed to `allow("", null)` instead to match the other 'optional' fields.

Comment (Rory-Powell) [reply packages/worker/src/api/routes/global/email.js]: just to confirm, can we send a request to this endpoint with those fields missing completely?
that's odd on optional, it usually works. Did you try it like this?
```ts
Joi.string().email().optional()
```

Comment (melohagan) [reply packages/worker/src/api/routes/global/email.js]: Yeah that's how I tried it, but it was already optional without it. The allow `null` means that null values can be sent through.

The following scenarios tested to work, the request went through:
 - Blank fields
 - JavaScript returning `undefined`
 - JavaScript returning `null`

Comment (Rory-Powell) [reply packages/worker/src/api/routes/global/email.js]: sorry just to confirm again on Blank fields, do you mean with the value `""` or with the field omitted completely?

Comment (melohagan) [reply packages/worker/src/api/routes/global/email.js]: both!

**Review by Rory-Powell — APPROVED** [review]
LGTM

**Pull request merged**